modules/starfish/2.0/src/starfish.py

```diff
@@ -0,0 +1,467 @@
+#!/usr/bin/env python3
```

Comment:
No, I added a function to throw an error if the packages for venn diagram drawing aren't available. Otherwise the script runs happily with no explanation as to why the venn wasn't produced.

Reply:
Can you document that change in the code if you haven't already?

Reply:
Documented in changelog.